ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXOpenCVVideoVisualizer.java

```diff
@@ -62,10 +62,15 @@ public void updateImageDimensions(int imageWidth, int imageHeight)
    }
 
    public void setImage(Mat mat)
+   {
+      setImage(mat, opencv_imgproc.COLOR_RGB2RGBA);
+   }
+
+   public void setImage(Mat mat, int inputColorFormat)
```

Comment:
I think it'd be good if the parameter name made it explicit that it should be the input color format to RGBA conversion. Maybe something like this? 
```suggestion
   public void setImage(Mat mat, int toRGBAConversion)
```

Or just add javadoc stating that :shrug:

Love this change though